tests/Unit/ParallelAlgorithms/LinearSolver/Multigrid/Test_MultigridAlgorithm.cpp

```diff
@@ -0,0 +1,128 @@
+// Distributed under the MIT License.
```

Comment:
It could be nice to have unit tests using the action-testing framework for any of the actions that can be tested on their own, so that any problems or changes that arise in particular actions can be more easily found and fixed.

Reply:
As mentioned above https://github.com/sxs-collaboration/spectre/pull/3249#discussion_r672615525 I didn't find the action tests for the internal actions of the parallel linear solvers particularly useful so far. I did add a fixup with action-tests for the public restriction actions though, so thanks for pointing this out!